src/field_theory/intermediate_field.lean

```diff
@@ -317,6 +321,37 @@ begin
     left, refl }
 end
 
+/-- Let `L/K` be a field extension and let `E, F` be intermediate fields with `E ≤ F`(where
+  `hEF` is a term of type `E ≤ F`). Then `intermediate_field.incusion hEF` is the `K`-algebra
+  homomorphism `E →ₐ[K] F` given by the inclusion. -/
+def inclusion {K L : Type*} [field K] [field L] [algebra K L]
+  {E F : intermediate_field K L} (hEF : E ≤ F):
+  E →ₐ[K] F :=
+{ to_fun := set.inclusion hEF,
```

Comment:
Is this just `subalgebra.inclusion hEF`?

Reply:
Sorry, that comment wasn't clear: What I mean is, we should still have this as its syntactically nicer, but it should be implemented as `subalgebra.inclusion`

Reply:
But since `subalgebra.inclusion` already exists, doesn't that prevent me from defining it in the `intermediate_field` file? Do you think it was a mistake to move the stuff from the PR into `subalgebra`?

Reply:
What I'm suggesting is:
```suggestion
def inclusion {K L : Type*} [field K] [field L] [algebra K L]
  {E F : intermediate_field K L} (hEF : E ≤ F):
  E →ₐ[K] F :=
subalgebra.inclusion hEF
```